make/test.mk

```diff
@@ -35,7 +35,18 @@ test-e2e: deploy-e2e e2e-run
 
 .PHONY: deploy-e2e
 deploy-e2e: INSTALL_OPERATOR=true
-deploy-e2e: build clean-e2e-files deploy-host deploy-members e2e-service-account setup-toolchainclusters
+deploy-e2e: build clean-e2e-files label-olm-ns deploy-host deploy-members e2e-service-account setup-toolchainclusters
+
+label-olm-ns: create-host-project create-member1 create-member2
```

Comment:
these makefile targets are after that called also inside of the deploy steps. Technically, it shouldn't be a problem, but I'm not 100% sure

Reply:
these makefile targets will print in error saying that the project(s) already exist, but that won't cause failure

Reply:
indeed, there will be an error because the project already exists, but it's ignored and the script goes on:
```
Error from server (AlreadyExists): project.project.openshift.io "toolchain-host-operator" already exists
make[1]: [create-project] Error 1 (ignored)
```